### ISSUE TITLE
Blogeintrag

### ISSUE THREAD

**Issue by milafrerichs** [codeformuenster/wahlprogramm-matrix#1]

**Comment by ubergesundheit** [comment]
@kirel Diesmal Lust, wo du ja nen afd branch aufgemacht hast?

**Comment by kirel** [comment]
Jo, allerdings ist das ne Aktion mit meinem neuen Arbeitgeber, darum in deren Blog. Ihr (wir) als Code for Münster werdet aber zitiert. :)
![code for munster 2016-04-02 11-04-37](https://cloud.githubusercontent.com/assets/9124/14225422/b65812d2-f8c2-11e5-932c-f346804acf35.png)

**Comment by ubergesundheit** [comment]
:+1: denke einen kurzen Eintrag der dann auf den anderen Post verlinkt werden wir dann auch machen!
